File: examples/BulkSendJobListExample.ts
import * as DropboxSign from "@dropbox/sign";

const bulkSendJobApi = new DropboxSign.BulkSendJobApi();

// Configure HTTP basic authorization: api_key
bulkSendJobApi.username = "YOUR_API_KEY";

// or, configure Bearer (JWT) authorization: oauth2
// bulkSendJobApi.accessToken = "YOUR_ACCESS_TOKEN";

const page = 1;
const pageSize = 20;

const result = bulkSendJobApi.bulkSendJobList(page, pageSize);
result.then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling Dropbox Sign API:");
  console.log(error.body);
});

File: examples/BulkSendJobListExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
    # config.access_token = "YOUR_ACCESS_TOKEN"
end

begin
    response = Dropbox::Sign::BulkSendJobApi.new.bulk_send_job_list(
        {
            page: 1,
            page_size: 20,
        },
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling BulkSendJobApi#bulk_send_job_list: #{e}"
end
